test/test_issue1003.py

```diff
@@ -105,8 +106,12 @@ def test_scenarios() -> None:
     ds1.base = base_three
 
     trix = ds1.serialize(format="trix", base=Namespace("http://two.org/"))
-    assert '<graph xml:base="http://one.org/">' in trix
-    assert '<graph xml:base="http://two.org/">' in trix
+
+    with pytest.raises(Exception):
+        assert '<graph xml:base="http://one.org/">' in trix
+    with pytest.raises(Exception):
+        assert '<graph xml:base="http://two.org/">' in trix
```

Comment:
Should the graphs not have bases set?

Reply:
All too likely to be an mistake on my part, I'll change the test and revert the method additions.